pybossa/syncer/project_syncer.py

```diff
@@ -43,8 +43,10 @@ class ProjectSyncer(Syncer):
     def sync(self, project):
         target = self.get_target(short_name=project.short_name)
         payload = self._build_payload(project, target)
+        is_new_project = False
 
         if not target:
+            is_new_project = True
```

Comment:
Is this line of code used? It appears to return on the next line, without using it. The above comment would address this as well.